js/apps/account-ui/test/account-security.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { test, expect } from "@playwright/test";
+import { login } from "./login";
+
+test("Check page heading", async ({ page }) => {
+  await login(page, "jdoe", "jdoe", "photoz");
+  await page
+    .getByRole("button", {
+      name: "Account security",
+    })
+    .click();
+
+  await page.waitForSelector(
+    '[data-testid="account-security/linked-accounts"]',
+  );
```

Comment:
According to the [Playwright documentation](https://playwright.dev/docs/api/class-page#page-wait-for-selector) this method is discouraged:

> Use web assertions that assert visibility or a locator-based [locator.waitFor()](https://playwright.dev/docs/api/class-locator#locator-wait-for) instead. Read more about [locators](https://playwright.dev/docs/locators).

It also seems like locators like `page.getByTestId()` will 'auto wait', so perhaps this can be omitted?

```suggestion
```

Reply:
thanks @jonkoops yes, you're right. I was trying to see if it would finally pass. Trying out different approaches. `waitForSelector ` is not going to be used.